### ISSUE TITLE
finished abnormally, OS return value: -11

### ISSUE THREAD

**Issue by Soumyadutta-basak** [ablab/spades#1072]
### Description of bug

Hey I have been trying to run spades
This is the command i used

spades.py --only-assembler -o SPADES_OUT -1 S17_Trimmed_R1.fq.gz -1 S46_Trimmed_R1.fq.gz -1 S62_Trimmed_R1.fq.gz -1 S72_Trimmed_R1.fq.gz -1 S81_Trimmed_R1.fq.gz -2 S17_Trimmed_R2.fq.gz -2 S46_Trimmed_R2.fq.gz -2 S62_Trimmed_R2.fq.gz -2 S72_Trimmed_R2.fq.gz -2 S81_Trimmed_R2.fq.gz

I got the error message as shown below ending the spades.py running:
== Error == system call for: "['/tools/spades/bin/spades-core', '/home/sandeeplab1/WGS_fungi/3_Assembly/SPADES_OUT/K21/configs/config.info']" finished abnormally, OS return value: -11

### spades.log

[spades.log](https://github.com/ablab/spades/files/10546087/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/10546091/params.txt)

### SPAdes version

SPAdes genome assembler v3.15.5

### Operating System

Ubuntu 22.04.1

### Method of SPAdes installation

manual 

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
How much free RAM is available for your SPAdes job? Does `spades.py --test` run properly?

**Comment by Soumyadutta-basak** [comment]
Yes i tried spades.py --test but its still showing the same error

= Error ==  system call for: "['/tools/spades/bin/spades-hammer', '/home/sandeeplab1/WGS_fungi/3_Assembly/spades_test/corrected/configs/config.info']" finished abnormally, OS return value: -11


[spades.log](https://github.com/ablab/spades/files/10553364/spades.log)
[params.txt](https://github.com/ablab/spades/files/10553365/params.txt)

**Comment by asl** [comment]
I'd suspect there is some I/O issue on the volume that your `/home` is located. Maybe it does not support memory-mapped files properly. I would suggest you to talk to your system administrator.

**Comment by Soumyadutta-basak** [comment]
I re-installed SPAdes and its working fine now. Thank you so much for helping.

**Comment by SowmyaPulapet** [comment]
Hi @Soumyadutta-basak , 
    Would you please tell me how did you solve this issue ? I am also facing the same issue.

**Comment by crockydavr** [comment]
> Hi @Soumyadutta-basak , Would you please tell me how did you solve this issue ? I am also facing the same issue.

I managed to solve this problem by compiling.  The manual describes in detail what is needed for it.

**Comment by SowmyaPulapet** [comment]
Oh okay, Thank you. @crockydavr
